setup-homebrew/action.yml

```diff
@@ -8,23 +8,23 @@ inputs:
   core:
     description: Update the `homebrew/core` tap.
     required: false
-    default: true
+    default: false
```

Comment:
Looks like CI in this repo also assumes this is `true`.

Reply:
CI will be from `brew update` picking up `HOMEBREW_NO_INSTALL_FROM_API`.

Reply:
Hmmm, ok I misread the error I was referring to then. But then I now don't know what this one's about:
https://github.com/Homebrew/actions/actions/runs/5467676415/jobs/9954295989?pr=393#step:8:11

Reply:
That's what by mean - that homebrew-core fetch is coming from the `brew update --auto` we are doing here.

Reply:
It is, however, not what I meant. I don't know why I always end up copying and pasting the wrong links, but this is the error I'm referring to:
```
Error: Failure while executing; `/usr/bin/git -C /home/linuxbrew/.linuxbrew/Homebrew describe --tags --abbrev --dirty` exited with 128. Here's the output:
```

Reply:
That's a test-bot bug introduced with https://github.com/Homebrew/homebrew-test-bot/commit/ce58be9afc9e4fb823e88a64a93c536834693378. We set a global gitconfig in setup-homebrew but that's no longer being respected.

Reply:
If we change `--local` to copy `~/.gitconfig` into new `HOME` it should fix the issue.